### PR TITLE
Clean empty lines

### PR DESCRIPTION
**Pull request by matiasbeckerle**
## Background

I took a look at the project and I saw many empty lines and I thought it would be nice to clean those up.

## Changes

- Cleaned up some unnecessary usings.
- Removed some empty lines.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,13 +1,9 @@
 ﻿using System;
-using System.Collections.Generic;
```
Comment (jrocaspana) [inline Program.cs:2]: are you sure we need to remove this?

File: Program.cs [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
-using System.IO;
```
Comment (jrocaspana) [inline Program.cs:3]: won't we need this in a future?

**Review by jrocaspana — COMMENTED** [review]
nice job! left some questions for you

File: Program.cs (hunk 1 above)
Comment (jrocaspana) [reply Program.cs:2]:
```sql
select * from table;
```

File: Program.cs [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
 
 namespace RentCar
```
Comment (jrocaspana) [inline Program.cs:3]: nice change!

**Comment by matiasbeckerle** [comment]
It was just for illustrative purposes.

**Pull request closed without merging**